lib/views/pages/home/posts/post_card/post_card.dart

```diff
@@ -0,0 +1,86 @@
+import "package:flutter/material.dart";
+import "package:proxima/models/ui/post_data.dart";
+import "package:proxima/views/pages/home/posts/post_card/comment_widget.dart";
+import "package:proxima/views/pages/home/posts/post_card/user_bar_widget.dart";
+import "package:proxima/views/pages/home/posts/post_card/votes_widget.dart";
+
+/// This widget is used to display the post card in the home feed.
+/// It contains the post title, description, votes, comments
+/// and the user (profile picture and username).
+class PostCard extends StatelessWidget {
+  static const postCardKey = Key("postCard");
+  static const postCardTitleKey = Key("postCardTitle");
+  static const postCardDescriptionKey = Key("postCardDescription");
+  static const postCardVotesKey = Key("postCardVotes");
+  static const postCardCommentsKey = Key("postCardComments");
+  static const postCardUserKey = Key("postCardUser");
+
+  final Post post;
+
+  const PostCard({
+    super.key,
+    required this.post,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final postBottomBar = Padding(
+      padding: const EdgeInsets.all(8),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          VotesWidget(key: postCardVotesKey, votes: post.votes),
+          InkWell(
+            customBorder: RoundedRectangleBorder(
+              borderRadius: BorderRadius.circular(20),
+            ),
+            //TODO: Implement the logic to navigate to the post
```

Comment:
We already have a navigation action below.

Reply:
Hello, I am not sure to understand why you mean by 

> We already have a navigation action below

In my code, clicking on the comment part of the bottom bar should be done using this `onTap: () => {},` to cover the entire space taken by the widget.

Reply:
The Inkwell widget in the Card already has an ontap function that navigates to the post. I don't understand why the postbottombar should do the same.

Reply:
Oh! I see ! It was to separate the click on the post from the click on the number of comments. It is purely for the UI 😄 

In terms of UI, it makes an InkWell response on the entire card (post + comment card) when you click on the post. And if you click on the comment number/ icon, you will see a small InkWell response only on the small button. 
😄 

https://github.com/ProximaEPFL/proxima/assets/53620532/5fd1c0a4-aecc-4b41-b897-1f9bfadfc839